contentcuration/contentcuration/frontend/accounts/router.js

```diff
@@ -0,0 +1,28 @@
+import VueRouter from 'vue-router';
+import Main from './pages/Main.vue';
+import Create from './pages/Create.vue';
+import ActivationSent from './pages/ActivationSent.vue';
+import ForgotPassword from './pages/ForgotPassword.vue';
+
+function pageRoute(path, component) {
+  return {
+    path,
+    name: component.name,
+    component: component,
+  };
+}
```

Comment:
by convention, set the route name to the component name

Reply:
Lovely idea